pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "evergreen.py"
-version = "3.13.1"
+version = "3.14.0"
```

Comment:
I figured this is appropriate because external usage changed, lmk if it's right!